coding-guides/cleverbot-integration.md

```diff
@@ -56,10 +56,10 @@ cleverbot = new Cleverbot;
 ...and put them with our discord definitions.
 
 ```javascript
-const Discord = require("discord.js");
+const { Client, Intents } = require("discord.js");
 const Cleverbot = require("cleverbot-node");
-const client = new Discord.Client({
-  intents: ["GUILDS", "GUILD_MESSAGES"]
```

Comment:
Why make the users enable a privileged intent? Shouldn't we encourage the usage of interactions and move away from messageCreate the way discord.js did?

Reply:
This was written way-way-way before privileged intents were a thing, we don't typically remove content unless it's actually causing more harm than good.  

i.e. selfbots/glitch/heroku/repl.it hosting, they all got the axe *as well as a ton of other stuff* because it was either really, really easy to do, or caused more harm.

Plus, we will be encouraging users to use interactions, but it still (yet at least) make message based commands completely obsolete.

You can still get the message content if a user mentions your bot, or DM's it so it's still perfectly valid.

Reply:
I disagree with this
Discord.js already completely detached itself from using legacy commands
They gave us up until april 2022, saying they expect all bots to migrate to slash commands
Since discord doesn't encourage it and discord.js already moved from legacy commands, I don't see why we should promote it and leave the viewers for a total rewrite the moment message content access gets deprecated

Reply:
That is discord.js, we are not discord.js, we do not need to follow the same conventions and standards as they do, we are a separate entity.

As I have said **repeatedly**, we will not be _**removing, or replacing**_ the legacy code, we will however be including slash command versions of the legacy code, and that will be prioritised over legacy commands due to the privileged intent.